core/src/main/java/io/parsingdata/metal/Shorthand.java

```diff
@@ -171,6 +171,8 @@ private Shorthand() {}
     public static ValueExpression fold(final ValueExpression values, final BinaryOperator<ValueExpression> reducer, final ValueExpression initial) { return foldRight(values, reducer, initial); }
     public static ValueExpression rev(final ValueExpression values) { return new Reverse(values); }
     public static ValueExpression exp(final ValueExpression base, final ValueExpression count) { return new Expand(base, count); }
```

Comment:
Urgh, apparently if I open a link in my mail app, it opens inside that app, in which these emoticons are not shown. :cry: